README.md

```diff
@@ -15,10 +15,10 @@ Django Privacy Analytics fully respects the 'Do Not Track' (DNT) header; if a re
 
 ### Access
 
-Currently, the dashboard is only visible to superusers. This will be configurable in a future version.
+Analyics dashboard can be controlled by user permissions with the `PermissionsRequiredMixin`
```

Comment:
Hmm, this is a bit unclear. Can we write out the exact permissions that Django Privacy Analytics uses to determine access control here? `PermissionsRequiredMixin` is just a way to check permissions, after all—can we list the permissions we're checking themselves?

Reply:
Right now it uses the basic permissions that Django automatically populates once we import the models, e.g., private_analytics.view_pageview, private_analytics.create_pageview, etc. Giving the option for them to implement more specific permissions which can then be updated via settings which I want to add